src/librustdoc/html/static/js/search.js

```diff
@@ -612,6 +606,34 @@ window.initSearch = function(rawSearchIndex) {
             onEach(crateAliases, pushFunc);
         }
 
+        // This function adds the given result into the provided `res` map if it matches the
+        // following condition:
+        //
+        // * If it is a "literal search" (`isExact`), then `lev` must be 0.
+        // * If it is not a "literal search", `lev` must be <= `MAX_LEV_DISTANCE`.
+        //
+        // `fullId` is the key used of the object we use for the `res` map.
+        //
+        // `id` is the index in both `searchWords` and `searchIndex` arrays for this element.
+        //
+        // `index` is used as a value when sorting results.
```

Comment:
"is used as a value when sorting results" doesn't tell me much. :-) Can you explain more? What type is it? How is it used in sorting?